### PR TITLE
Call cob-shutdown when the power button is pressed

### PR DESCRIPTION
**Pull request by ipa-nhg**
Added cob-stop-core script as @ipa-mdl suggested

### CODE REVIEW

**Comment by ipa-nhg** [comment]
Test on cob4-7: OK

File: scripts/cob-shutdown [hunk 1]
```diff
@@ -17,11 +17,11 @@ $robot_name-s1
...
+#cob-stop-core
 
-while /opt/ros/indigo/env.sh rostopic list > /dev/null; do
-	sleep 1
-done
+#while /opt/ros/indigo/env.sh rostopic list > /dev/null; do
+#	sleep 1
+#done
 
 for client in $client_list; do
```
Comment (ipa-fmw) [inline scripts/cob-shutdown]: does this really wait for all pc to shutdown __before__ the robot is turned off?

Please make sure that if the base pc is the last one which shuts down.

**Comment by ipa-nhg** [comment]
test on cob4-2 : OK

File: scripts/cob-shutdown (hunk 1 above)
Comment (ipa-nhg) [inline scripts/cob-shutdown]: done: https://github.com/ipa320/setup_cob4/pull/46/commits/a14f8b3c72b262fc59c8e54854a680422e175c70

File: scripts/cob-shutdown [hunk 2]
```diff
@@ -24,16 +23,26 @@ $robot_name-b1"
...
-		echo "command return an error (error code: $ret), aborting..."
-		#exit 1
-	fi
-	echo ""
+    echo "-------------------------------------------"
+    echo "Executing <<"shutdown">> on $client"  
+    echo "-------------------------------------------"
+    echo ""
+    ssh $client "sudo shutdown now -P 0"
+    if [ $ret != 0 ] ; then
```
Comment (ipa-mdl) [inline scripts/cob-shutdown]: ret is undefined here

Comment (ipa-nhg) [inline scripts/cob-shutdown]: done https://github.com/ipa320/setup_cob4/pull/46/commits/2c964cda60b5fad0a28efcfc117b4ab5c5d54c28

**Comment by ipa-nhg** [comment]
test on cob4-7 : OK

**Comment by ipa-fmw** [comment]
tested on cob4-2: OK

**Pull request merged**